provision/acc_provision/apic_provision.py

```diff
@@ -221,28 +237,51 @@ def valid_tagged_resource(self, tag, system_id, tenant):
     def clean_tagged_resources(self, system_id, tenant):
         tags = collections.OrderedDict([])
         tags_path = "/api/node/mo/uni/tn-%s.json" % (tenant,)
-        tags_path += "?query-target=subtree&target-subtree-class=tagInst"
+        apic_version = self.get_apic_version()
```

Comment:
Should we always do the check for tagged resources as well? This might perhaps manifest in the case where upgrade has not been done and previous install had tagged resources?

Reply:
@snaiksat I agree with you. It's better to do tagInst always regard less of the version

Reply:
@snaiksat  Are you referring to check both TagInst, TagAnnotation resources regardless of APIC version? I don't see a problem in doing that.

@apoorva11029  You can take out GetVersion code if you agree.